index.html

```diff
@@ -230,23 +230,47 @@ <h3 class="teamsubtitle">Lorem ipsum dolor sit amet proin gravida nibh vel velit
             </div>  
             
             <div class="mainworks">
```

Comment:
rename class name like 
```suggestion
            <div class="main-works">
```